lib/Runtime/Base/ThreadContextInfo.cpp

```diff
@@ -418,7 +418,12 @@ ThreadContextInfo::SetValidCallTargetForCFG(PVOID callTargetAddress, bool isSetV
 #ifdef _CONTROL_FLOW_GUARD
     if (IsCFGEnabled())
     {
+#ifdef _M_ARM
+        AssertMsg(IS_16BYTE_ALIGNED((uintptr_t)callTargetAddress & ~0x1), "callTargetAddress is not 16-byte aligned?");
+        AssertMsg(((uintptr_t)callTargetAddress & 0x1) != 0, "on ARM we expect the thumb bit to be set on anything we use as a call target");
```

Comment:
>callTargetAddress [](start = 30, length = 17)

This should be before IS_16BYTE_ALIGNED assert.

Reply:
Re-ordered.